kubernetes-monitoring/mgmtagent_helm/values.yaml

```diff
@@ -26,4 +30,4 @@ kubernetesCluster:
    # Kubernetes cluster name
    name:
    # Kubernetes cluster Namespace to monitor
```

Comment:
Can you update the documentation (comment for this line).  "Kubernetes cluster namespace(s) to monitor.  This can be a comma-separated list of namespaces or '*' to monitor all the namespaces"

Can you check if spaces in the comma-separated list is allowed or not?  If it is not allowed then we should have that stated in the comment as well.

Reply:
added the updated comments.

Note: Namespaces with spaces are allowed.